examples/rhfe_dual.py

```diff
@@ -65,7 +65,7 @@ def get_romol_conf(mol):
 
 # for RHFE we need to insert the reference ligand first, before inserting the
 # decoupling ligand
-minimized_coords = minimizer.minimize_4d(romol_a, system, host_coords, ff, box)
+minimized_coords = minimizer.minimize_host_4d([romol_a], system, host_coords, ff, box)
```

Comment:
This doesn't fix this script (IE it doesn't run), but at least it isn't try to call a non-existent API

Reply:
Don't worry about it - I will fix this later.